thunder/executors/cudnnex.py

```diff
@@ -383,9 +414,6 @@ def _make_cudnn_sdpa_backward_graph(
     b, h, s_q, _ = query.size
     _, _, _, d_v = value.size
 
-    # cuDNN < 9.0.0 might produce nan gradients for sequence length < 64
```

Comment:
Removed because cudnn now internally checks this condition and has the checker not claim sdpa operator if not supproted.